controllers/provisioners/eks/update.go

```diff
@@ -111,6 +111,11 @@ func (ctx *EksInstanceGroupContext) Update() error {
 		return errors.Wrap(err, "failed to update scaling group")
 	}
 
+	if updated {
+		// requeue after scaling group update occurs to refresh cache
```

Comment:
Is this something we want to be durable long-term, or just for now to fix the loop? Feels like we should update cache eventually rather than relying on the next reconcile to do it.

Reply:
I find from past experience with this & other controllers it's much easier to manage the state using this pattern of having short reconciles and requeue when you need to, otherwise you either find yourself making excessive amount of calls or  working on semi-accurate data.

Excessive amount of API calls is not that bad actually now that we use https://github.com/keikoproj/aws-sdk-go-cache 
we used to get throttled heavily before on clusters with many InstanceGroups/other.